src/index.js

```diff
@@ -4,13 +4,16 @@ import "./index.css";
 import { App } from "./App";
 import reportWebVitals from "./reportWebVitals";
 import { RecipesProvider } from "./modules/recipes/RecipesProvider";
+import { BrowserRouter } from "react-router-dom";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
-    <RecipesProvider>
-      <App />
-    </RecipesProvider>
+    <BrowserRouter>
+      <RecipesProvider>
+        <App />
```

Comment:
I'd recommend to move UserContext from App component here so that all the contexts are in the same place